.gitlab/find-gh-base-ref.sh

```diff
@@ -40,74 +40,44 @@ if [[ $(git log --pretty=oneline origin/master..HEAD | wc -l) -eq 1 ]]; then
   exit 0
 fi
 
-# In GitLab: we have no reference to the base branch or even the PR number.
-# We have to find it from the current branch name, which is defined in
-# CI_COMMIT_REF_NAME.
-if [[ -z "${CI_COMMIT_REF_NAME}" ]]; then
-  echo "CI_COMMIT_REF_NAME is not set, not running in GitLab CI?" >&2
-  exit 1
-fi
-
-# In GitLab, CI_PROJECT_NAME is set, otherwise, set it for testing.
-export CI_PROJECT_NAME="${CI_PROJECT_NAME:-dd-trace-java}"
+get_distance_from_merge_base() {
+  local candidate_base="$1"
+  merge_base_sha=$(git merge-base "$candidate_base" HEAD)
+  distance=$(git log --pretty=oneline "$merge_base_sha".."$CURRENT_HEAD_SHA" | wc -l)
```

Comment:
What about using `git rev-list` instead? Ref: https://git-scm.com/docs/git-rev-list
```suggestion
  distance=$(git rev-list "$merge_base_sha".."$CURRENT_HEAD_SHA")
```